src/cli/onefuzz/api.py

```diff
@@ -568,29 +571,48 @@ def _dbg_windows(
         proxy = "*:" + REPRO_SSH_FORWARD if bind_all else REPRO_SSH_FORWARD
         with ssh_connect(repro.ip, repro.auth.private_key, proxy=proxy):
             dbg = ["cdb.exe", "-remote", "tcp:port=1337,server=localhost"]
-            if debug_command:
-                dbg_script = [debug_command, "qq"]
-                with temp_file("db.script", "\r\n".join(dbg_script)) as dbg_script_path:
-                    dbg += ["-cf", _wsl_path(dbg_script_path)]
-
+            while True:
```

Comment:
Maybe we should stop after N tries, if it hasn't worked after a few minutes it _probably_ won't and that way people don't waste time thinking it might eventually work. Maybe N=10/20/30?

Reply:
I'll set it to 10